src/main/groovy/io/openliberty/tools/gradle/tasks/InstallFeatureTask.groovy

```diff
@@ -110,7 +120,15 @@ class InstallFeatureTask extends AbstractServerTask {
         }
 
         def dependencyFeatures = getDependencyFeatures()
-        def serverFeatures = getServerDir(project).exists() ? util.getServerFeatures(getServerDir(project)) : null
+
+        def serverFeatures = null;
+
+        // if DevMode provides a server directory parameter use that for finding the server features
+        if (serverDirectoryParam != null) {
+            serverFeatures = util.getServerFeatures(new File(serverDirectoryParam));
+        } else if (getServerDir(project).exists()) {
+            serverFeatures = util.getServerFeatures(getServerDir(project));
+        }
 
         Set<String> featuresToInstall = InstallFeatureUtil.combineToSet(pluginListedFeatures, dependencyFeatures, serverFeatures)
 
```

Comment:
We should have a way to override the first parameter in `util.installFeatures()` to be `true` so that the licenses are accepted.  This is because WebSphere Liberty features need license acceptance.  We could allow another optional command line parameter for `acceptLicense`, and set it to `true` from DevTask.

Reply:
This isn't needed since the build.gradle file should have 
```
liberty {
    server {
        features {
            acceptLicense = true
        }
    }
}
```
 as shown in example 4 of https://github.com/OpenLiberty/ci.gradle/blob/master/docs/installFeature.md#installfeature-task